### ISSUE TITLE
[flux-dsl] `aggregateWindow` yet to be supported?

### ISSUE THREAD

**Issue by zhongjiewu** [influxdata/influxdb-client-java#107]
I don't see aggregateWindow in the flux-dsl module. Is it yet to be supported?

I understand that it is *similar* to window + yield, just asking as this is convenient.

**Comment by bednar** [comment]
Hi @zhongjiewu,

Currently we don't have a `aggregateWindow` in our `DSL` module. PR is welcome ;)

Regards

**Comment by bednar** [comment]
Hi @zhongjiewu,

The issue is fixed in [1.8.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/11). 

You could use something like:

```java
Flux flux = Flux
    .from("telegraf")
    .aggregateWindow(10L, ChronoUnit.SECONDS, "mean");
```

If you would like to use a preview version use our snapshot build from repository: https://oss.sonatype.org/content/repositories/snapshots/

Regards
